content/intro/1.md

```diff
@@ -28,8 +28,10 @@ at https://dgraph.io/docs/graphql/overview/ this tutorial is exclusive for DQL
 (GraphQL+-). {{% /notice %}}
 
 The tutorial is divided into modules. You can access the table of contents at
-any time by clicking "A Tour of Dgraph" in the top left and can move backwards
-and forwards using the right and left arrows at the bottom of this pane.
+any time by clicking on the hamburger icon in the bottom left and can move backwards
+and forwards using the **Back** and **Next** buttons at the bottom of this pane, or
+by pressing the <kbd>Left Arrow</kbd> and <kbd>Right Arrow</kbd> keys of your
+keyboard.
```

Comment:
I'd suggest:

"You can access the table of contents at any time by clicking on the context menu ("sandwich" icon) in the bottom left corner of the screen; you can also navigate using the **Back** and **Next** buttons at the bottom of this pane, or  by pressing the <kbd>Left Arrow</kbd and <kbd>Right Arrow</kbd> keys of your keyboard.